src/server.rs

```diff
@@ -27,70 +34,80 @@ impl Server {
     pub fn new() -> Self {
         Self {
             router: Arc::new(Router::new()),
+            headers: Arc::new(DashMap::new()),
         }
     }
 
     pub fn start(&mut self, py: Python, port: u16) {
-        // starts the server and binds to the mentioned port
-        // initialises the tokio async runtime
-        // initialises the python async loop
+        if STARTED
+            .compare_exchange(false, true, SeqCst, Relaxed)
```

Comment:
What does `SeqCst` mean in AtomicBool?

Reply:
Is the strongest ordering https://doc.rust-lang.org/std/sync/atomic/enum.Ordering.html#variants